### PR TITLE
Add openshift-applier inventory

### PR DESCRIPTION
**Pull request by jacobsee**

### CODE REVIEW

File: .applier/params/omp-frontend-buildconfig [hunk 1]
```diff
@@ -0,0 +1 @@
+GIT_URL=https://github.com/jacobsee/open-management-portal-frontend.git
```
Comment (mcanoy) [inline .applier/params/omp-frontend-buildconfig]: Using jacobsee GitHub repo. Intended? Happens in a number of spots

File: README.md [hunk 2]
```diff
@@ -9,6 +9,7 @@ The following environment variables are available:
 | Name | Example Value | Required |
```
Comment (mcanoy) [inline README.md]: Is there a place in the doc that explains how to use the applied?

File: .applier/templates/omp-frontend-buildconfig.yml [hunk 3]
```diff
@@ -0,0 +1,40 @@
+apiVersion: v1
+kind: Template
```
Comment (mcanoy) [inline .applier/templates/omp-frontend-buildconfig.yml]: I’d be interested in adding the imagestream service and route to the template.

File: .applier/inventory/group_vars/all.yml [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+- object: omp-frontend
+  content:
+  - name: imagestream
+    file: '{{ inventory_dir }}/../files/omp-frontend-imagestream.yml'
+  - name: buildconfig
+    template: '{{ inventory_dir }}/../templates/omp-frontend-buildconfig.yml'
+    params: '{{ inventory_dir }}/../params/omp-frontend-buildconfig'
+  - name: deploymentconfig
+    template: '{{ inventory_dir }}/../templates/omp-frontend-deploymentconfig.yml'
+    params: '{{ inventory_dir }}/../params/omp-frontend-deploymentconfig'
```
Comment (mcanoy) [inline .applier/inventory/group_vars/all.yml]: Generally we show a preference for Params  forms cars. For me I like having them in your file but I’m ok with this approach

File: .applier/templates/omp-frontend-buildconfig.yml [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+    labels:
+      app: omp-frontend
+    name: omp-frontend
+  spec:
+    failedBuildsHistoryLimit: 5
+    nodeSelector: null
+    output:
+      to:
+        kind: ImageStreamTag
+        name: omp-frontend:latest
```
Comment (mcanoy) [inline .applier/templates/omp-frontend-buildconfig.yml]: Would be good to Param this. Both and and version

File: .applier/templates/omp-frontend-buildconfig.yml [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+    source:
+      git:
+        ref: master
+        uri: "${GIT_URL}"
+      type: Git
+    strategy:
+      sourceStrategy:
+        from:
+          kind: ImageStreamTag
+          name: nodejs:10
```
Comment (mcanoy) [inline .applier/templates/omp-frontend-buildconfig.yml]: Would be good to param this. Ideally run this on Apache or nginx

Comment (jacobsee) [reply .applier/templates/omp-frontend-buildconfig.yml]: Note that changing to Apache or Nginx will require a change in how the frontend is actually serving content. As it currently stands, the app is using server-side rendering in JavaScript to try to improve client experience (and honestly as a PoC for how we might want to do this in the future), but we can switch it to a static content mode and use one of those servers if desired.
I'll tackle other suggested changes here shortly.

File: README.md (hunk 2 above)
Comment (jacobsee) [reply README.md]: Added!

**Pull request merged**